src/Explorer/Explorer.tsx

```diff
@@ -875,9 +918,35 @@ export default class Explorer {
       handleError(error, "Explorer/onNewNotebookClicked");
       throw new Error(error);
     }
+    const isPhoenixEnabled = NotebookUtil.isPhoenixEnabled();
+    if (isPhoenixEnabled) {
+      if (isGithubTree) {
+        async () => {
+          await this.allocateContainer();
```

Comment:
>allocateContainer [](http://example.com/codeflow?start=21&length=17)

can we check how Google Collab works?

if someone wants to just open new notebook , do we want to allocate - even if they dont run it?

again - how many users will create new without running  - may not be large... but will be good to see what is collab's model?

Reply:
Understood, if we can move allocate to cell level that could be great. Will discuss and work on this in next PR.